src/resmom_win/start_exec.c

```diff
@@ -850,7 +850,7 @@ bld_wenv_variables(char *name, char *value)
 	int	i;
 	char    str_buf[MAXPATHLEN+1] = {0};
 
-	if ((*name == '\0') || (*name == '\n'))
+	if ((name == NULL) || (name == '\0') || (name == '\n'))
```

Comment:
ooooh - big mistake you missed the (star) *name in the second and third comparison rendering those checks useless

Reply:
The previous checks are fine, all you need to do is add a if (!name) || ... to it - rest is already good